modules/gis/src/lib/geojson-to-binary.js

```diff
@@ -216,7 +216,7 @@ function secondPass(features, firstPassData = {}, options = {}) {
 
   for (const feature of features) {
     const geometry = feature.geometry;
-    const properties = feature.properties;
+    const properties = feature.properties || {};
```

Comment:
According to the declared typings, this _shouldn't_ be necessary
https://github.com/visgl/loaders.gl/blob/bb450ef5e6f03cdf43269abd5e866274af379ef1/modules/gis/src/types.d.ts#L39

But it's possible not all generated data matches the typing exactly. Ideally we'd be able to move the code to TS at some point in the future.

Reply:
I've added this shot circuit eval to avoid setting `properties` as null when the geojson contains this value, as long as the GeoJSON format allows it (https://tools.ietf.org/html/rfc7946):
> A Feature object has a member with the name "properties".  The value of the properties member is an object (any JSON object or a JSON null value).

Also in case of having another falsy value, even if it doesn't fit the standard.

Reply:
Good point, I forgot `properties` can be `null`.